app/views/conferences/_lodging.html.haml

```diff
@@ -12,22 +12,26 @@
           = @conference.venue.city
       %p.lead
         We recommend these affordable lodging accommodations for your visit.
-  - @conference.lodgings.each_slice(3) do |slice|
-    .row.row-centered
-      - slice.each do |lodging|
-        .col-md-4.col-sm-4.ticket.col-centered.col-top
-          .thumbnail
-            - unless lodging.picture?
-              %p.text-center
-                %i.fa.fa-home.fa-5x
-            - else
+
+  .row.row-centered{ style:"display: flex; flex-wrap: wrap" }
+    - @conference.lodgings.each do |lodging|
+      .col-md-4.col-sm-4.col-centered.col-top{ style:"display:flex;" }
+        .thumbnail
+          - if not lodging.picture?
```

Comment:
Here you should use ! instead of not, but I don't understand why your workflow is
`if not sth ... else...` instead of 
`if sth ... else ...`